### PR TITLE
Schema 3.0.0

### PR DESCRIPTION
**Pull request by fzadrazil**
* **Breaking:** Remove obsolete `HSLayers.Layer.WMS` and `OpenLayers.Layer.Vector` layer types
* support WMTS layer type

### CODE REVIEW

**Comment by jirik** [comment]
> I would also like to target this #10 in the PR. Does anyone has any idea of the meaning or possible usage of the "urls" parameter?

Not used in Layman.

File: schema.json [hunk 1]
```diff
@@ -456,17 +456,18 @@
             "description": "Big type of layer. Currently all raster layers are wms layers and Vector layers are subdivided by protocol property",
             "default": "",
             "enum": [
-              "HSLayers.Layer.WMS",
-              "OpenLayers.Layer.Vector",
+              "WMS",
```
Comment (jmacura) [inline schema.json:459]: Duplicate item `"WMS"`

**Review by jmacura — APPROVED** [review]

**Review by FilipLeitner — APPROVED** [review]

**Comment by fzadrazil** [comment]
Any relevant reason to have `hs.format` there?

https://github.com/hslayers/map-compositions/blob/b273d6a312757a6b1c3aee5f1e1a96e7ed84dd18/schema.json#L539-L541

Wouldn't be better to have just eg. 'WFS'?

File: schema.json [hunk 2]
```diff
@@ -456,17 +456,18 @@
...
               "WMS",
               "WFS",
```
Comment (index-git) [inline schema.json:460]: "ClassName: WFS" is never used, I suggest to remove it also.

**Review by index-git — CHANGES_REQUESTED** [review]

File: schema.json (hunk 2 above)
Comment (jmacura) [reply schema.json:460]: Not used in HSLayers either. But wouldn't it be future-proof to keep the WFS classname as an allowed value?

**Review by index-git — APPROVED** [review]
